groovy-language-server/src/main/java/com/palantir/groovylanguageserver/GroovycWrapper.java

```diff
@@ -116,6 +131,61 @@ public Path getWorkspaceRoot() {
     }
 
     @Override
+    public Map<String, Set<SymbolInformation>> getReferences() {
+        return references;
```

Comment:
After thinking about it more, I still think this is the nicer solution, because otherwise if the client calls `findReferences` with code that has not compiled successfully it will get an empty list instead of an error.